store/views.py

```diff
@@ -77,6 +107,30 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    template_name = 'store/loaned_books.html'
+    get_data = request.POST
+    obj=BookCopy.objects.get(book__id=get_data['bid'])
+    obj.borrower=None
+    obj.borrow_date=None
+    obj.status=True
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.